lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
 import ImportStatement from './ImportStatement';
 
+const STYLE_SIDE_EFFECT = 'side-effect';
```

Comment:
Same thought as above. If the environment tells me it has side-effects, should I sort it to the top regardless of whether it also has exports or not?